src/test/data/bitcoin-util-test.json

```diff
@@ -136,15 +157,17 @@
      "in=5897de6bd6027a475eadd57019d4e6872c396d0716c4875a5f1a6fcfdf385c1f:0",
      "outaddr=0.18:13tuJJDR2RgArmgfv6JScSdreahzgc4T6o",
      "outdata=4:54686973204f505f52455455524e207472616e73616374696f6e206f7574707574207761732063726561746564206279206d6f646966696564206372656174657261777472616e73616374696f6e2e"],
-    "output_cmp": "txcreatedata1.json"
+    "output_cmp": "txcreatedata1.json",
+    "description": "Creates a new transaction with one input, one address output and one data output. Output in json"
```

Comment:
nit: lines from here down can be changed to use "(output in json)" to match the above.